bootstrap/config/kfctl_default.yaml

```diff
@@ -28,7 +28,12 @@ components:
   - centraldashboard
   - admission-webhook  
   - jupyter-web-app
-  - katib
+  - katib-db
```

Comment:
What's the benefit to break katib into:
  - katib-db
  - katib-manager
  - katib-controller
  - katib-ui
?

Reply:
@kkasravi mentioned this in https://github.com/kubeflow/kubeflow/pull/3463#issuecomment-501846718.

This allows groupings of related microservices and the use of kustomize features like prefixes and labels.